### PR TITLE
change dcos-diagnostics nssm config (DCOS_OSS-5697) 

### PR DESCRIPTION
**Pull request by OleksandrBielov**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5697](https://jira.mesosphere.com/browse/DCOS_OSS-5697) Fix NSSM configuration file for Mesos agent, Diagnostic and Telegraf EE / fix diagnostics nssm config

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by OleksandrBielov** [comment]
@mesosphere-mergebot changelog-not-required reason

**Comment by OleksandrBielov** [comment]
@mesosphere-mergebot request-review @vadymzagorodko

**Comment by OleksandrBielov** [comment]
@mesosphere-mergebot request-review @DominikDary

**Comment by OleksandrBielov** [comment]
@mesosphere-mergebot request-review @jongiddy

File: packages/dcos-diagnostics/windows.build.ps1 [hunk 1]
```diff
@@ -18,5 +18,6 @@ Copy-Item -Path "$SRC_DIR/dcos-diagnostics.exe" -Destination "$env:PKG_PATH/bin/
 
 New-Item -ItemType Directory "$env:PKG_PATH/etc"
 Copy-Item "c:/pkg/extra/dcos-diagnostics.nssm.j2" "$env:PKG_PATH/etc/"
+Copy-Item "c:/pkg/extra/servicelist.txt" "$env:PKG_PATH/bin/"
```
Comment (jongiddy) [inline packages/dcos-diagnostics/windows.build.ps1]: Why do we need this in the `bin` directory, which normally only contains programs that can be executed? This looks like it should be in `etc`.

Comment (DominikDary) [reply packages/dcos-diagnostics/windows.build.ps1]: I reviewed my documents from my former investigation ([see](https://github.com/DominikDary/diagnostics-win) )and there the `servicelist.txt` file had be be place close to the binary.
This my can be changed for the future, but for now I think it is fine how it is.

Comment (DominikDary) [reply packages/dcos-diagnostics/windows.build.ps1]: the `servicelist.txt` file is used by diagnostics to contact the windows services and get their status which will be displayed in dcos ui.

**Comment by jongiddy** [comment]
@mesosphere-mergebot changelog-not-required Part of Windows changes

**Pull request merged**